deps/hashicorp/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:latest
+FROM alpine:3.14
```

Comment:
what about just using `3` and not worrying about the `minor` ?

Reply:
I tend to not rely on automatic changes even if they are minor since they can happen at any time with no plannification and breaks unexpectcly